assets/admin/entry.js

```diff
@@ -0,0 +1 @@
+import 'sylius/bundle/AdminBundle/Resources/private/entry';
```

Comment:
I guess we should follow this approach just to be consistent everywhere: https://github.com/Sylius/Sylius-Standard/commit/e97cb7c578ac071ab6cb4438b252be1dbd85abe5